source/administration/bucket-replication.rst

```diff
@@ -256,6 +256,9 @@ workers operating on that queue. MinIO continuously works to replicate and
 remove objects from the queue while scanning for new unreplicated objects to
 add to the queue. 
 
+MinIO persists failed replications to disk and re-queues the replication repeatedly until the replication succeeds.
```

Comment:
Nit for later - which disk? is it erasure coded? We're inviting some questions here.

Maybe hide it as something like "MinIO queues failed replication operations and retries those operations until replication succeeds".